main_ghost.py

```diff
@@ -25,32 +23,39 @@
 'Always use a database. What\'s a JSON?',
 'Powered by Electricity',
 'Who still writes docs in 2018?',
-"First normal form? What does that mean?",
-"By using a relational database but with nonrelational practices we get the worst of both worlds!",
-"I haven\'t paid attention or read any comments, therefor it\'s impossible to understand!",
-"Don\'t use that! Oh, you\'re asking why? Well... just don\'t it.",
-"I don\'t wanna explain, just Google it.",
-"What are cogs?",
-"This is MY project. You\'re just freeloaders.",
-"You've got three weeks to fix EVERYTHING.",
-"No-one agrees? Too bad! My idea it is.",
-"The next version will be written in Java only!"
+'First normal form? What does that mean?',
+'By using a relational database with nonrelational practices we get the worst of both worlds!',
+'I haven\'t paid attention or read any comments, therefore it\'s impossible to understand!',
+'Don\'t use that! Oh, you\'re asking why? Well... just don\'t it.',
+'I don\'t wanna explain, just Google it.',
+'What are cogs?',
+'This is MY project. You\'re just freeloaders.',
+'You\'ve got three weeks to fix EVERYTHING.',
+'No-one agrees? Too bad! My idea it is.',
+'The next version will be written in Java only!'
 ]
 
 import discord
 import random
 import asyncio
 
 # Import config data
+# Imports go (folder name).(file name)
 import story_time.cc_creation as creation_messages
 import story_time.powerup as secret_messages
 from config import welcome_channel, game_master, dead_participant, frozen_participant, administrator, peasant
-from config import ghost_prefix as prefix
+from config import ww_prefix as prefix

```

Comment:
This is ghost bot - why are you importing `ww_prefix`?

Reply:
Erm - good question. Lemme fix that.

Reply:
I don't think you've fixed this, have you?

Reply:
*coughs intensely* Oh! Look at that! A commit!